README.md

```diff
@@ -23,84 +21,79 @@ Currently the operator installs the following products:
-- [oc](https://docs.okd.io/3.11/cli_reference/get_started_cli.html#cli-reference-get-started-cli) version v3.11+
-- Access to an Openshift v4.2.0+ cluster
+- [oc](https://docs.okd.io/latest/cli_reference/openshift_cli/getting-started-cli.html) version v4.6+
+- Access to an Openshift v4.6.0+ cluster
 - A user with administrative privileges in the OpenShift cluster
-- AWS account with permissions to create S3 buckets
 
-After installation, the following commands must be run to avoid a known [issue](https://github.com/matryer/moq/issues/98) related to the Moq package:
-```
+After installation, the following commands must be run to avoid a known issue related to the Moq package:
+```shell
 make code/compile
 go install github.com/matryer/moq
 ```
 
-## Local Setup
+## Using `ocm` for installation of RHMI
+
+If you want to test your changes on a cluster, the easiest solution would be to spin up OSD 4 cluster using `ocm`.
+See [here](https://github.com/integr8ly/delorean/tree/master/docs/ocm) for an up to date guide on how to do this.
+
 
-Download the integreatly-operator project:
+## Local Development
+
+### 1. Clone the integreatly-operator
 ```sh
 mkdir -p $GOPATH/src/github.com/integr8ly
 cd $GOPATH/src/github.com/integr8ly
 git clone https://github.com/integr8ly/integreatly-operator
 cd integreatly-operator
 ```
 
-If the cluster is not already prepared for the integreatly-operator, you will need to do the following:
+### 2. Prepare your cluster
+
+If you are working against a fresh cluster it will need to be prepared using the following. 
+Ensure you are logged into a cluster by `oc whoami`.
+Include the `INSTALLATION_TYPE` if you haven't already exported it. Options are: 
+- managed (RHMI)
+- managed-api (RHOAM)
 ```
-make cluster/prepare/project
-make cluster/prepare/crd
-make cluster/prepare/smtp
+INSTALLATION_TYPE=<managed/managed-api> make cluster/prepare/local
 ```
 
-* 3scale requires AWS S3 bucket credentials for storage. The bucket should have all public access turned off.
-
-Currently this secret (`threescale-blobstorage-<installation-name>`) is created with dummy credentials by the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator), in the namespace the integreatly operator is deployed into. In order for this feature to work, these credentials should be replaced:
-    * _bucketName_: The name of the AWS bucket
-    * _bucketRegion_: The AWS region where the bucket has been created
-    * _credentialKeyID_: The AWS access key
-    * _credentialSecretKey_: The AWS secret key
+### 3. Configuration options
 
-You can use this command to replace S3 credentials in 3Scale secret:
-```sh
-oc process -f deploy/s3-secret.yaml -p AWS_ACCESS_KEY_ID=<YOURID> -p AWS_SECRET_ACCESS_KEY=<YOURKEY> -p AWS_BUCKET=<YOURBUCKET> -p AWS_REGION=eu-west-1 -p NAMESPACE=<integreatly-operator-namespace> -p NAME=threescale-blobstorage-<installation-name> | oc replace -f -
-```
+`USE_CLUSTER_STORAGE` - true or false \
```

Comment:
Let's flesh out this section a table with some information about what each command does. We use these two heavily in local dev.